src/kendo.data.js

```diff
@@ -1149,6 +1149,12 @@ var __meta__ = { // jshint ignore:line
             },
             isnotnull: function(a) {
                 return "(" + a + " != null)";
+            },
+            isnullorempty: function(a) {
+                return "!" + a;

```

Comment:
It's ok, maybe I just did not put the best description to pull request. There should be mentioned related issue/feat request to be all clear.